dynamiqs/time_array.py

```diff
@@ -389,49 +402,43 @@ def __add__(self, other: ArrayLike | TimeArray) -> TimeArray:
             return NotImplemented
 
 
-class ModulatedTimeArray(TimeArray):
```

Comment:
I guess you've inverted the order of `ModulatedTimeArray` and `CallableTimeArray` in the file and it's a bit too much for git to track changes properly. Can you re-order with `class ModulatedTimeArray` first and then `class CallableTimeArray` (to see the actual changes to these classes, and to match the `__all__` and documentation order).